nixos/modules/services/web-servers/apache-httpd/moodle.nix

```diff
@@ -0,0 +1,935 @@
+{ config, lib, pkgs, serverInfo, php, ... }:
+
+with lib;
+
+let
+
+  httpd = serverInfo.serverConfig.package;
+
+  version24 = !versionOlder httpd.version "2.4";
+
+  allGranted = if version24 then ''
+    Require all granted
+  '' else ''
+    Order allow,deny
+    Allow from all
+  '';
+
+  moodleConfig = pkgs.writeText "config.php"
+    ''
+<?php
+///////////////////////////////////////////////////////////////////////////
+//                                                                       //
```

Comment:
Since this is a generated file (which presumably won't be read by users), are these huge commented sections necessary?